### DISCUSSION TITLE
public key

### DISCUSSION THREAD

**Discussion by farzanehnematpour** [farzanehnematpour/Tafe24S1_14#1] [General]
please send your public key here.

**Comment by OscarCastle573** [comment]
Here is the key
---- BEGIN SSH2 PUBLIC KEY ----
Comment: "rsa-key-20240401"
AAAAB3NzaC1yc2EAAAADAQABAAABAQCbqaC0auRCunHsVZwICjkZy8tEpzidTFQv
AXfE5iZfgb2Du5RTK8R94e1ZtAyzGdPbFHtT+3pgE5NFWAfW1qEVHGzeGRwgk75F
fJkyhs4KPHujy2VnDx/HrKfbGMzczZsKv8Z9Pwc7C9ZFTZXgoiMauWNojtX+U0XY
/E0ZzeD/jQ/v61CuLUFhs9ni5Cnt5qY/p9u1rIrLS71ehlgTOgMtl+XOGkvj7yZe
xECv889pQVYswtfV3Npb/2uk/0Qw6T1g06jkW+ygtUL1ecT+Yb7+WWt0WYwO8ATb
tLUwQxREo7STcWUMM2JR1opgi7fRleDa9yfw5QYMIWXZXPfnI47r
---- END SSH2 PUBLIC KEY ----

**Comment by OscarCastle573** [comment]
Do you need the passphrase too?

**Comment by OscarCastle573** [comment]
Okay, hopefully this one's right then
AAAAB3NzaC1yc2EAAAADAQABAAABAQCbqaC0auRCunHsVZwICjkZy8tEpzidTFQvAXfE5iZfgb2Du5RTK8R94e1ZtAyzGdPbFHtT+3pgE5NFWAfW1qEVHGzeGRwgk75FfJkyhs4KPHujy2VnDx/HrKfbGMzczZsKv8Z9Pwc7C9ZFTZXgoiMauWNojtX+U0XY/E0ZzeD/jQ/v61CuLUFhs9ni5Cnt5qY/p9u1rIrLS71ehlgTOgMtl

**Comment by farzanehnematpour** [comment]
not working. you sure you are copying from key generator? it should start with ssh-rsa

**Comment by OscarCastle573** [comment]
Well what happens if you add that infront of the second code, does it work then, otherwise I did copy from the correct area in the key generator

**Reply by farzanehnematpour** [reply]
I tried. Didn't work

**Comment by farzanehnematpour** [comment]
make sure you scroll down while selecting the code. My key is longer something like 6 or 7 lines

**Comment by farzanehnematpour** [comment]
This my public key:
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7JZ4CAwBgRScEGQYya7ibig+03ZxZBE1aNc5Yd9+A4/uRa07zEAaMFZKhAWY91BZqs2ry0pRoidd6Zjj40Baf/3uHqAXOxEtJwf44Df1yKidB/6f/1lMAv7Qc7nM3R0PvWzAFK6uX/55lp14kPlksCE2msim0Pet0wV2vfabK1PKL6U/qHesK9ShYfGx+tkxDdB0B4bTpzS7yOjFNxcKo4WgvWKMA3Eqskf/4RiqJkE+hyuI6H3m83ntG0J5oWQ7EMMYtOT7ILo8KjPAySh78QOg3XpRp50Xl9QseOHlPWkhvxz6rR2fppGtR7PzFi0s2sjYvk+NMUgnCN7qBsJ4X rsa-key-20240317

**Comment by OscarCastle573** [comment]
Okay, lets try this,
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCbqaC0auRCunHsVZwICjkZy8tEpzidTFQvAXfE5iZfgb2Du5RTK8R94e1ZtAyzGdPbFHtT+3pgE5NFWAfW1qEVHGzeGRwgk75FfJkyhs4KPHujy2VnDx/HrKfbGMzczZsKv8Z9Pwc7C9ZFTZXgoiMauWNojtX+U0XY/E0ZzeD/jQ/v61CuLUFhs9ni5Cnt5qY/p9u1rIrLS71ehlgTOgMtl+XOGkvj7yZexECv889pQVYswtfV3Npb/2uk/0Qw6T1g06jkW+ygtUL1ecT+Yb7+WWt0WYwO8ATbtLUwQxREo7STcWUMM2JR1opgi7fRleDa9yfw5QYMIWXZXPfnI47r rsa-key-20240401

**Comment by farzanehnematpour** [comment]
Great. I added your key

**Comment by OscarCastle573** [comment]
Nice
